tests/runner.py

```diff
@@ -1084,7 +1084,8 @@ def ccshared(src, linkto=[]):
            'a: loaded\na: b (prev: (null))\na: c (prev: b)\n')
 
     self.set_setting('RUNTIME_LINKED_LIBS', [])
-    self.emcc_args += ['--embed-file', '.@/']
+    for libname in ['liba.js', 'libb.js', 'libc.js']:
```

Comment:
Should probably be  `+ so` rather than `.js`.

Reply:
Or maybe:

```
for libname in ['liba', 'libb', 'libc']:
  self.emcc_args += ['--embed-file', libname + '.so']
```

Reply:
Oh right, yeah, this runs in wasm mode too.